.github/workflows/release-library.yaml

```
@@ -0,0 +1,59 @@
+        default: "patch"
+        options:
+          - rc
+          - patch
+          - minor
+          - major
+concurrency:
+  group: release
+  cancel-in-progress: false
+jobs:
+  build-library:
+    name: Library ${{ matrix.architecture }}
```

actionlint (1.7.12)

[error] property "architecture" is not defined in object type {}

(expression)